docs/src/docker.md

```diff
@@ -145,3 +145,45 @@ The image will be tagged as `playwright:localbuild-focal` and could be run as:
 ```
 docker run --rm -it playwright:localbuild /bin/bash
 ```
+
+## (Experimental) Playwright Test Docker Integration
+* langs: js
+
+Playwright Test now ships an **experimental** docker integration.
+With this integration, **only** browser binaries are running inside a docker container,
```

Comment:
```suggestion
With this integration, browser binaries are running **only** inside a docker container,
```

Reply:
I'd disagree with this change since it changes the semantics of the sentence!

My point was to highlight that unlike the regular docker approach that mounts the whole project into docker & executes all tests & test runner & browser binaries inside, this approach keeps test runner on the host system, while only launching browsers inside the container.